servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -4720,6 +5188,78 @@
         }
       }
     },
+    "ClusterUpgradeHealthPolicy": {
+      "type": "object",
+      "description": "The upgrade health policy of the cluster",
+      "properties": {
+        "MaxPercentDeltaUnhealthyNodes": {
```

Comment:
any min/max value constraints? Since the property name has percent in it, i assume 0-100. Can you please update the constraints?